### ISSUE TITLE
Deprovision service bus on closing a cluster

### ISSUE THREAD

**Issue by isaacabraham** [mbraceproject/MBrace.Azure#136]
I think that we should consider removing the service bus queue after deprovisioning a cluster. We're using the standard service bus package which has a cost associated with it without using it for anything. The cost is not massive to be fair - around 10eur a month but nonetheless someone can delete a cluster and be left with running costs anyway.

Any objection for me changing the management library to do this?

**Comment by eiriktsarpalis** [comment]
This has already been [implemented](https://github.com/mbraceproject/MBrace.Azure/blob/master/src/MBrace.Azure.Management/Management.fs#L91), but it is disabled by default. In principle, an mbrace cluster is a superset of any particular deployment; multiple deployments can be part of the same cluster either at the same time or successively. The nice thing about this is that computations can be suspended when no workers are running.

**Comment by isaacabraham** [comment]
Yes. But we both know some people are particular about running costs of clusters - particularly if hidden. I know what you're saying about suspend / resume, but how often is that going to be happening? I would suggest that most people spin up a cluster and spin it down when they're done with it.
